protocol-samples/http-client-to0-sample/src/main/java/org/fido/iot/sample/To0ClientApp.java

```diff
@@ -278,8 +278,7 @@ private void run(String[] args)
     Composite rvi = ovh.getAsComposite(Const.OVH_RENDEZVOUS_INFO);
 
     List<String> paths = RendezvousInfoDecoder.getHttpDirectives(
-        rvi,
-        Const.RV_OWNER_ONLY);
+        rvi, Const.RV_DEV_ONLY);
```

Comment:
I believe that the filter in getHttpDirectives should be Const.RV_OWNER_ONLY for To0ClientApp.